tasks/kyu5.py

```diff
@@ -0,0 +1,32 @@
+"""Implementation of Fibonacci function and Fibonacci product function."""
+
+
+def fib(num):
+    """
+    Returns n-th element from a Fibonacci sequence.
+    """
+    first, second = 0, 1
+    for _ in range(2, num - 1):
+        first, second = second, first + second
+    return second
+
+
+def fib_product(product):
+    """
+    Finds two consistent Fibonacci numbers, being given their possible product.
+    In case it is - returns a list with [fib(n), fib(n+1), True],
+    otherwise [fib(n), fib(n+1), False].
+    """
+    current_value, num = 0, 0
```

Comment:
You can write this like (_current_value = num = fib1 = fib2 = 0_)

Reply:
I prefer it to look like this because I think that different kinds of variables should be in different lines for better code-reading.